Exercise_3/MainActivity.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Android.App;
+using Android.Widget;
+using Android.OS;
+using Android.Views;
+
+namespace Exercise_3
+{
+    [Activity(Label = "Exercise_3", MainLauncher = true)]
+    public class MainActivity : Activity
+    {
+        [InjectView(Resource.Id.edFirstCofficient)] private EditText edFirstCofficient;
+        [InjectView(Resource.Id.edSecondCofficient)] private EditText edSecondCofficient;
+        [InjectView(Resource.Id.edThirdCofficient)] private EditText edThirdCofficient;
+        [InjectView(Resource.Id.tvResultTitle)] private TextView tvResultTitle;
+        [InjectView(Resource.Id.tvResult)] private TextView tvResult;
+        [InjectOnClick(Resource.Id.btnSolve)]
+        private void Solve(object sender, EventArgs e)
+        {
+            tvResultTitle.Visibility = ViewStates.Invisible;
+            tvResult.Text = "";
+            if (edFirstCofficient.Text.Equals("") ||
```

Comment:
Change it to `string.IsNullOrEmpty(edFirstCofficient.Text)`...

Reply:
done